### PR TITLE
docs: adding resources for mobile, reorganizing Other resources

### PR DESCRIPTION
**Pull request by janchild**
This PR updates the **Designing > Other resources** page by: 

- Creating a new section for mobile Sketch resources, and
- Adding the Dark theme, the grid template for mobile, and patterns and examples

### CODE REVIEW

**Review by tay1orjones — APPROVED** [review]
looks good! 👍

File: src/pages/designing/design-resources/index.mdx [hunk 1]
```diff
@@ -196,4 +198,46 @@ for PAL sites or any other sites using the Gatsby theme.
...
+or Android project underway. Install the grid template, be inspired by the
+patterns and examples, and get started with either the Light or Dark theme.
+
+<Row className="resource-card-group">
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Light theme for mobile"
+    href="sketch://add-library/cloud/s/a3343128-adb6-489c-9e62-709d89ba76e9"
+    actionIcon="launch">
```
Comment (laurenmrice) [inline src/pages/designing/design-resources/index.mdx:213]:
```suggestion
    actionIcon="download">
```

File: src/pages/designing/design-resources/index.mdx [hunk 2]
```diff
@@ -196,4 +198,46 @@ for PAL sites or any other sites using the Gatsby theme.
...
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Dark theme for mobile"
+    href="sketch://add-library/cloud/s/1f59f590-6915-47b0-bf06-6fd66209b3b3"
+    actionIcon="launch">
```
Comment (laurenmrice) [inline src/pages/designing/design-resources/index.mdx:221]:
```suggestion
    actionIcon="download">
```

**Review by jillianhowarth — APPROVED** [review]

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks good!

- Just wanted to change the two icons to `download` for the direct downloads for the light and dark theme mobile kits.

Two other things I noticed:
- The Gatsby theme kit resource card should also get a `download` icon.
- In the "Extensions" section, there is still a resource card for Carbon Mobile sketch kit.

**Comment by janchild** [comment]
Thanks for the eagle eyes, @laurenmrice! Just updated!

**Review by laurenmrice — APPROVED** [review]
🚀 👏

**Pull request merged**